src/test/e2e/helpers/test-helper.ts

```diff
@@ -179,7 +179,16 @@ async function createApp(
         server: {
             unleashUrl: 'http://localhost:4242',
         },
-        ...customOptions,
+        ...{
+            ...customOptions,
+            experimental: {
+                ...(customOptions?.experimental ?? {}),
+                flags: {
+                    strictSchemaValidation: true,
+                    ...(customOptions?.experimental?.flags ?? {}),
+                },
+            },
+        },
```

Comment:
Enable strict schema validation unless it's been explicitly disabled. This applies to all app setup methods.